sphinx_scylladb_theme/static/js/main.bundle.js

```diff
@@ -1,2 +1,2 @@
=0;e<s;e++)i[e]&&i[e].finish&&i[e].finish.call(this);delete n.finish}))}}),j.each(["toggle","show","hide"],(function(t,e){var n=j.fn[e];j.fn[e]=function(t,i,o){return null==t||"boolean"==typeof t?n.apply(this,arguments):this.animate(ce(e,!0),t,i,o)}})),j.each({slideDown:ce("show"),slideUp:ce("hide"),slideToggle:ce("toggle"),fadeIn:{opacity:"show"},fadeOut:{opacity:"hide"},fadeToggle:{opacity:"toggle"}},(function(t,e){j.fn[t]=function(t,n,i){return this.animate(e,t,n,i)}})),j.timers=[],j.fx.tick=function(){var t,e=0,n=j.timers;for(oe=Date.now();e<n.length;e++)(t=n[e])()||n[e]!==t||n.splice(e--,1);n.length||j.fx.stop(),oe=void 0},j.fx.timer=function(t){j.timers.push(t),j.fx.start()},j.fx.interval=13,j.fx.start=function(){re||(re=!0,le())},j.fx.stop=function(){re=null},j.fx.speeds={slow:600,fast:200,_default:400},j.fn.delay=function(t,e){return t=j.fx&&j.fx.speeds[t]||t,e=e||"fx",this.queue(e,(function(e,n){var o=i.setTimeout(e,t);n.stop=function(){i.clearTimeout(o)}}))},function(){var t=b.createElement("input"),e=b.createElement("select").appendChild(b.createElement("option"));t.type="checkbox",m.checkOn=""!==t.value,m.optSelected=e.selected,(t=b.createElement("input")).value="t",t.type="radio",m.radioValue="t"===t.value}();var he,pe=j.expr.attrHandle;j.fn.extend({attr:function(t,e){return Q(this,j.attr,t,e,arguments.length>1)},removeAttr:function(t){return this.each((function(){j.removeAttr(this,t)}))}}),j.extend({attr:function(t,e,n){var i,o,r=t.nodeType;if(3!==r&&8!==r&&2!==r)return void 0===t.getAttribute?j.prop(t,e,n):(1===r&&j.isXMLDoc(t)||(o=j.attrHooks[e.toLowerCase()]||(j.expr.match.bool.test(e)?he:void 0)),void 0!==n?null===n?void j.removeAttr(t,e):o&&"set"in o&&void 0!==(i=o.set(t,n,e))?i:(t.setAttribute(e,n+""),n):o&&"get"in o&&null!==(i=o.get(t,e))?i:null==(i=j.find.attr(t,e))?void 0:i)},attrHooks:{type:{set:function(t,e){if(!m.radioValue&&"radio"===e&&z(t,"input")){var n=t.value;return t.setAttribute("type",e),n&&(t.value=n),e}}}},removeAttr:function(t,e){var n,i=0,o=e&&e.match(q);if(o&&1===t.nodeType)for(;n=o[i++];)t.removeAttribute(n)}}),he={set:function(t,e,n){return!1===e?j.removeAttr(t,n):t.setAttribute(n,n),n}},j.each(j.expr.match.bool.source.match(/\w+/g),(function(t,e){var n=pe[e]||j.find.attr;pe[e]=function(t,e,i){var o,r,s=e.toLowerCase();return i||(r=pe[s],pe[s]=o,o=null!=n(t,e,i)?s:null,pe[s]=r),o}}));var ve=/^(?:input|select|textarea|button)$/i,me=/^(?:a|area)$/i;function ge(t){return(t.match(q)||[]).join(" ")}function ye(t){return t.getAttribute&&t.getAttribute("class")||""}function be(t){return Array.isArray(t)?t:"string"==typeof t&&t.match(q)||[]}j.fn.extend({prop:function(t,e){return Q(this,j.prop,t,e,arguments.length>1)},removeProp:function(t){return this.each((function(){delete this[j.propFix[t]||t]}))}}),j.extend({prop:function(t,e,n){var i,o,r=t.nodeType;if(3!==r&&8!==r&&2!==r)return 1===r&&j.isXMLDoc(t)||(e=j.propFix[e]||e,o=j.propHooks[e]),void 0!==n?o&&"set"in o&&void 0!==(i=o.set(t,n,e))?i:t[e]=n:o&&"get"in o&
```

Comment:
no idea what this change is

Reply:
This is the minified js. For more information, see https://sphinx-theme.scylladb.com/stable/contribute/contribute-theme.html#building-the-frontend